build.assets/build-package.sh

```diff
@@ -80,6 +80,12 @@ if [ ! $(command -v curl) ]; then
     exit 2
 fi
 
+# check that tar is installed
+if [ ! $(command -v tar) ]; then
+    echo "tar must be installed"
+    exit 11
```

Comment:
`exit 1` ?

Reply:
The idea originally was that there'd be a different non-zero exit code from each possible error in the script so we could potentially use that to work out the reason for failure. I just continued that trend onwards :man_shrugging:

Reply:
Ahh, I didn't look below and assumed that `11` was a typo instead of `1`.
Makes sense now